examples/express-composition/index.js

```diff
@@ -17,6 +17,8 @@ if (require.main === module) {
         // useful when used with OpenAPI-to-GraphQL to locate your application
         setServersFromRequest: true,
       },
+      // Use the LB4 application as a route. It should not be listening.
+      noListen: true,
```

Comment:
I find the option name `noListen` potentially confusing, because it's introducing double negative (`noListen: false`). Have you considered using `listen` instead? Or perhaps `listenOnStart`.